### ISSUE TITLE
Blank tab after 'Facebook Containers' menu option on Firefox for Android

### ISSUE THREAD

**Issue by robpearman** [mozilla/contain-facebook#508]
- Facebook Container Version: 2.0.1
- Operating System + Version: Android 9
- Firefox Version: 68.0.2
- Other installed Add-ons + Version + Enabled/Disabled-Status: Default v1.0, Enabled // OpenH264 Video Codec, unknown version, enabled

### Actual behavior
Select 'Facebook Container' from the Firefox menu (three dots top-right of the screen) and a new tab is opened (URL moz-extension://11956130-f923-4c42-885e-9e3fbf4d1a6e/panel.html) but this page/tab is completely blank

### Expected behavior
Unclear, just hoping for some feedback that the add-on is working as it should

### Steps to reproduce
1. Install Firefox and Firefox Container add-on
2. Open the menu (three dots top-right of the screen)
3. Select 'Facebook Container' from the menu

### Notes
I'm assuming that the add-on is supported on the Android version of Firefox and that it is working correctly but get no feedback to confirm that, if visiting a site that has cookies blocked by the add-on on the desktop site (e.g. https://www.igeeksblog.com/how-to-use-reachability-on-iphone-x/ or a billion others)

Thanks for the add-on, I love it on desktop :)

**Comment by maxxcrawford** [comment]
Unfortunately, this add-on is _not__ currently supported for Android. I'm marking this as a enhancement/wontfix. Thanks!
